src/build_tools.rs

```diff
@@ -133,13 +139,25 @@ impl SchemaError {
     }
 
     pub fn from_val_error(py: Python, error: ValError) -> PyErr {
-        match error {
-            ValError::LineErrors(line_errors) => {
-                let details = pretty_line_errors(py, line_errors);
-                SchemaError::new_err(format!("Invalid Schema:\n{details}"))
+        let validation_error = ValidationError::from_val_error(py, "Schema".to_object(py), error, None);
```

Comment:
it's a bit ugly to go `ValidationError -> PyErr -> ValidationError`.

Probably easiest if I have a play changing it?

Reply:
sure